chain/network/src/peer/peer_actor.rs

```diff
@@ -1196,7 +1217,14 @@ impl actix::Handler<stream::Frame> for PeerActor {
     type Result = ();
     #[perf]
     fn handle(&mut self, stream::Frame(msg): stream::Frame, ctx: &mut Self::Context) {
-        let _span = tracing::trace_span!(target: "network", "handle", handler = "bytes").entered();
+        let _span = tracing::trace_span!(
```

Comment:
Any reason to not reuse `handler_trace_span!(target: "network", msg_len = msg.len(), peer = %self.peer_info)` here?

I acknowledge that `handler = "bytes"` will change to `handler = "Frame"`, but if I was looking at the traces and was familiar with how other handlers are instrumented, I’d have a hard time making the connection that this is a special case.